docs/integrations/business-intelligence/connecting-to-metabase.md

```diff
@@ -35,26 +33,26 @@ You are now ready to run Metabase with your Firebolt databases.
-| **Host**                       |  `api.app.firebolt.io`                                                                                                                                                                            |                                                        
-| **Port**                       |  `8123`                                                                                                                                                                                           |                                                        
-| **Database name**              |  The name of your database in Firebolt                                                                                                                                                            |                                                        
-| **Username**                   | The username of your Firebolt account with access privileges to the specified database                                                                                                                                                                          |                                                       
-| **Password**                   |  The password associated with the username of your Firebolt account                                                                                                                                                                           |                                                       
-| **Additional JDBC parameters** |  Can be left blank. Any additional parameters for Firebolt can be added here. For example, the following paramater specifies a Firebolt engine to use for queries from Metabase: `engine=<my_database_engine_1>` |
+| Field                          | Description                                                                  | 
+| ------------------------------ | ---------------------------------------------------------------------------- |
+| **Name**                       | A title to refer to your database in Metabase. For simplicity, we recommend using the same name as your Firebolt database. | 
+| **Host**                       | The Firebolt host to connect to. It is always `api.app.firebolt.io`.         |              
+| **Port**                       | The port to use for the connection. It is `8123`.                            |              
+| **Database name**              | The name of the Firebolt database to connect to.                             |              
+| **Username**                   | The name of the Firebolt user to connect with, e.g. `janedoe@mycompany.com`. | 

```

Comment:
Username is the terminology we use in the topic and it's in some UI dialogs too if I recall correctly. It matters because this is the email. Love that you added the example. Also, avoid abbreviations (difficult for non-native speakers): "The username to connect with, for example, `janedoe@mycompany.com`".